pkg/xds/envoy/clusters/pass_through_cluster_configurer.go

```diff
@@ -0,0 +1,26 @@
+package clusters
+
+import (
+	envoy_api "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+)
+
+func PassThroughCluster(name string) ClusterBuilderOpt {
+	return ClusterBuilderOptFunc(func(config *ClusterBuilderConfig) {
+		config.Add(&PassThroughClusterConfigurer{
+			Name: name,
+		})
+		config.Add(&AltStatNameConfigurer{})
+		config.Add(&TimeoutConfigurer{})
+	})
+}
+
+type PassThroughClusterConfigurer struct {
+	Name string
```

Comment:
And here also the public field. It looks like all configurers have public fields. Do we need it?